spec/models/book_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  context 'validity of Book' do
+    it 'is valid with valid attributes' do
+      book = Book.create(
```

Comment:
Here you can use the `let` instance method provided by RSpec to structure you're tests. It uses more of RSpec's DSL to make unit tests still be Ruby but also be different to your actual code.

What you've done is also valid.I prefer the below, the tests scale better and it's more inline with the RSpec communities' intent.

```ruby
context 'validations' do
  # Define an instance variable available to all tests within this context. 
  # Each test gets it's own instantiation, they don't share it
  let(:book) do # this is still just a plain old instance method accepting a symbol and a block.
    Book.create(
      author_name: author_name,
      rating: rating
    )
  end
  let(:author_name) { 'pq' }
  let(:rating) { 5 }
  
  it 'valid when valid' do
    expect(book).to be_valid # our instance method created by let(:book)
  end

  context 'rating missing' do
    # Just like other variable scoping in Ruby, here the duplicate let block takes precedence
    # over the one above
    let(:rating) {} # nil
   
    it 'is invalid' do
      expect(book).to_not be_valid
    end
  end
end
```